versioned_docs/version-1.2.x/surrealql/statements/create.mdx

```diff
@@ -25,58 +25,143 @@ CREATE [ ONLY ] @targets
 ;
 ```
 
-## Example usage
+## Creating a Table Record
 
-The following query shows example usage of this statement.
+When using the create statment the name following the `CREATE` keyword is the Table name, and following the the `:` is the record identifier. The two togther form the full [record ID](/docs/surrealdb/surrealql/datamodel/ids) which can be used to query the created data or by using the [`SELECT`](/docs/surrealdb/surrealql/statements/select) statment. The record identifier can be a string or a number. If no record identifier is specified, a random identifier will be generated.
+
+:::note
+__Note:__ You can also assign the recordID to be [randomly generated](/docs/surrealdb/surrealql/datamodel/ids#generating-record-ids) 
+:::
+
+The query below will create a new person table and also create a new record with a random ID.
 
 ```surql
--- Create a new record with a random id
-CREATE person SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+-- Create a new table
+CREATE person
+```
+
+```bash title="Response"
+[
+    {
+        "id": "person:2vvgzt6m24s952yiy7x8"
+    }
+]
+```
 
+Alternatively, you can specify the record identifier of the record you want to create.
+
+```surql
 -- Create a new record with a specific numeric id
-CREATE person:100 SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+CREATE person:100
+```
 
--- Create a new record with a specific string id
-CREATE person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+The above will create a new record with the ID `person:100`.
 
--- Create just a single record
--- Using the ONLY keyword, just an object for the record in question will be returned.
--- This, instead of an array with a single object.
-CREATE ONLY person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+```bash title="Response"
+[
+	{
+		"id": "person:100"
+	}
+]
 ```
 
-Instead of specifying record data using the `SET` clause, it is also possible to use the `CONTENT` keyword to specify the record data using a SurrealQL object.
+It is also possible to specify the ID of the record you want to create using a string or any of the supported formats for [record IDs](/docs/surrealdb/surrealql/datamodel/ids).
+
+
+
+## Adding Record Data
+
+When creating a record, you can specify the record data using the `SET` clause, or the `CONTENT` keyword. The `SET` clause is used to specify the record data using a list of key-value pairs, while the `CONTENT` keyword is used to specify the record data using a SurrealQL object. The `CONTENT` keyword is useful when the record data is already in the form of a SurrealQL object.
```

Comment:
We really have keywords. But that has an* as well 😂 

 You can do `create create`
 but you can't do `create create set set ="set"`